src/D2L.CodeStyle.Analysis/MutabilityInspector.cs

```diff
@@ -125,7 +125,7 @@ HashSet<ITypeSymbol> typeStack
 
 				{
 					foreach( ISymbol member in type.GetMembers() ) {
-						if( IsMemberMutableRecursive( member, typeStack ) ) {
+						if( !member.IsStatic && IsMemberMutableRecursive( member, typeStack ) ) {

```

Comment:
A type with only static members has no state. My terminology probably conflates `type` with `instance of type`, but we really only care about the latter. 

Example:


```csharp
public class Foo {
    public static readonly Foo Default = new Foo();
}
```
`Foo` is safe here.

Why do we care? So that we can be smarter about something like this:

```csharp
public class PlainTextLangterm {

    public static readonly PlainTextLangterm Default = new PlainTextLangterm("Default");

    private readonly string _value;
    public PlainTextLangterm( string value ) { _value = value; }
}
```